src/app.js

```diff
@@ -21,5 +23,16 @@ app.use(favicon(__dirname + '/public/favicon.ico'));
 
```

Comment:
nitpick but these should be flipped and just serve from public:
app.use(favicon(__dirname + '/public/favicon.ico'));
app.use('/public', express.static('public'));

Reply:
like this?
```
app.use('/public', express.static('public'));
app.use(express.static('public')); 
```